### PR TITLE
Add `raw` field to the transaction

### LINKED ISSUE #1481: GraphQL - expose raw transaction data
**Describe what you want implemented** 
Add a graphQL field to return the raw transaction binary 

**Describe "Why" this is needed**
This is required by the wallet to get the full transaction history

### PR DESCRIPTION
**Pull request by herr-seppia**
Resolves #1481

### CODE REVIEW

File: pkg/gql/query/types.go [hunk 1]
```diff
@@ -171,6 +175,36 @@ var Hex = graphql.NewScalar(graphql.ScalarConfig{
...
+			return b64.StdEncoding.EncodeToString(value)
+		default:
+			return nil
+		}
+	},
+	// ParseValue parses GraphQL variables from `string` to `[]byte`.
+	ParseValue: func(value interface{}) interface{} {
+		switch value := value.(type) {
+		case string:
+			bytes, _ := b64.StdEncoding.DecodeString(value)
```
Comment (goshawk-3) [inline pkg/gql/query/types.go:195]: If we can't return the err, maybe we can just log it as an error.

File: pkg/gql/query/transactions.go [hunk 2]
```diff
@@ -80,6 +81,7 @@ func newQueryTx(tx core.ContractCall, blockHash []byte, timestamp int64, blockhe
 	qd.GasPrice = decoded.Fee.GasPrice
 
 	qd.GasSpent = tx.GasSpent()
+	qd.Raw = tx.StandardTx().Data
```
Comment (goshawk-3) [inline pkg/gql/query/transactions.go:84]: Could you update the gql README with an example usage of the new field?

**Review by goshawk-3 — APPROVED** [review]
LGTM, just two nitpicks.

File: pkg/gql/query/types.go (hunk 1 above)
Comment (herr-seppia) [reply pkg/gql/query/types.go]: Removed as we don't need to parse it at all

File: pkg/gql/query/transactions.go (hunk 2 above)
Comment (herr-seppia) [reply pkg/gql/query/transactions.go:84]: Added to the example with the full supported fields list

**Pull request merged**